### ISSUE TITLE
"sql server create" does not support specifying server version (e.g. to specify a V12 sql server)

### ISSUE THREAD

**Issue by mcrsw** [Azure/azure-xplat-cli#1837]
If you run:
`azure sql server create --defaultFirewallRule username p@55w0rd "West Europe"`
you get by default a V2 server.

I was hoping then to be able to run something like the following, to create a new V12 SQL server:
`azure sql server create --defaultFirewallRule --version 12.0 username p@55w0rd "West Europe"`

However, I get the error message:
`error: unknown option '--version'`

**Comment by markcowl** [comment]
@mcrsw thank you for reporting this issue, assigning to the appropriate team

**Comment by metaskills** [comment]
Indeed... being able to create a specific version would really help. Is there a way to upgrade the version after you create it? From my searching, I could not find that. Even if there was, it is not a synchronous task. So that means it is even more critical being able to create with a specific version.
